packages/swing-store/test/test-import.js

```diff
@@ -160,6 +164,23 @@ const importTest = test.macro(async (t, mode) => {
     [0, 2, 5, 8],
   );
 
+  // and confirm compressed and uncompressed transcript items
+  const itemRows = [
+    ...db.prepare('SELECT * FROM transcriptItems ORDER BY position').iterate(),
+  ];
+  t.deepEqual(itemRows, [
+    { vatID: 'v1', position: 8, item: 'load-snapshot', incarnation: 1 },
+    { vatID: 'v1', position: 9, item: 'delivery3', incarnation: 1 },
+  ]);
+  const compressedSpanRows = [
+    // prettier-ignore
+    ...db.prepare('SELECT * FROM transcriptCompressedSpans ORDER BY startPos').iterate(),
+  ];
+  t.deepEqual(
+    compressedSpanRows.map(sr => sr.startPos),
+    tsCompressedStarts,
+  );
+
```

Comment:
Note to self: I'm ok with not having this test also decompress the `compressedSpanRows`, because the clause just above this one used `dump()` to read out all the items through an API that's pretty close to `readSpan`, so we already have confidence that the implementation is somehow capable of providing all the data it's supposed to. This clause here only needs to assert that the data is compressed, rather than discrete, and checking `startPos` is sufficient for that.